requirements-minimal.txt

```diff
@@ -10,6 +10,7 @@ ipython
 Jinja2>=3.1.2
 lockfile
 moto
+prometheus-client==0.20.0
```

Comment:
we should remove this pin from requirement-minimal - leaving it in here means "any version that is not 0.20.0 will not work"

i.e., this line should just have the dependency name